src/apiUtility.js

```diff
@@ -0,0 +1,92 @@
+var _ = require('./utility');
+
+function buildPayload(accessToken, data, jsonBackup) {
+  if (_.isType(data.context, 'object')) {
```

Comment:
I think this is because `data.context` is required to be a string per our API spec. I think this is the related issue from node_rollbar: https://github.com/rollbar/node_rollbar/issues/73